sympy/stats/crv_types.py

```diff
@@ -1107,6 +1110,78 @@ def Dagum(name, p, a, b):
 
     return rv(name, DagumDistribution, (p, a, b))
 
+#-------------------------------------------------------------------------------
+# Davis distribution -----------------------------------------------------------
+
+class DavisDistribution(SingleContinuousDistribution):
+    _argnames = ('b', 'n', 'mu')
+
+    set = Interval(0, oo)
+
+    @staticmethod
+    def check(b, n, mu):
+        _value_check(b > 0, "Scale parameter b must be positive.")
+        _value_check(n > 1, "Shape parameter n must be above 1.")
+        _value_check(mu > 0, "Location parameter mu must be positive.")
+
+    def pdf(self, x):
+        b, n, mu = self.b, self.n, self.mu
+        dividend = b**n*(x - mu)**(-1-n)
+        divisor = (exp(b/(x-mu))-1)*(gamma(n)*zeta(n))
+        return dividend/divisor
+
+
+def Davis(name, b, n, mu):
+    r""" Create a continuous random variable with Davis distribution.
+
+    Explanation
+    ===========
+
+    The density of Davis distribution is given by
+
+    .. math::
+        f(x; \mu; b, n) := \frac{b^{n}(x - \mu)^{1-n}}{ \left( e^{\frac{b}{x-\mu}} - 1 \right) \Gamma(n)\Zeta(n)}
```

Comment:
Small fix:
```
f(x; \mu; b, n) := \frac{b^{n}(x - \mu)^{1-n}}{ \left( e^{\frac{b}{x-\mu}} - 1 \right) \Gamma(n)\zeta(n)}
```

Reply:
The easiest way to chdck whether it works is to put it in a text cell of a Jupyter notbeook enclosed by `$$`